documentation/configuration.md

```diff
@@ -1613,6 +1613,7 @@ An action that collects live metrics for the process that the collection rule is
 |---|---|---|---|---|---|---|
 | `IncludeDefaultProviders` | bool | false | Determines if the default counter providers should be used. | `true` | | |
 | `Providers` | [EventMetricsProvider](api/definitions.md#eventmetricsprovider)[] | false | The array of providers for metrics to collect. | | | |
```

Comment:
nit: we changed this for EventMetricsConfiguration to, "Array of **counter** providers for metrics to collect" - just if we're aiming for consistency